### PR TITLE
BitfocusAS.Companion version 2.4.2.4911

### PR DESCRIPTION
**Pull request by froskemann-erik**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95704)

### CODE REVIEW

File: manifests/b/BitfocusAS/Companion/2.4.2.4911/BitfocusAS.Companion.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: BitfocusAS.Companion
+PackageVersion: 2.4.2.4911
```
Comment (Trenly) [inline manifests/b/BitfocusAS/Companion/2.4.2.4911/BitfocusAS.Companion.installer.yaml:5]:
```suggestion
PackageVersion: 2.4.2
```

File: manifests/b/BitfocusAS/Companion/2.4.2.4911/BitfocusAS.Companion.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: BitfocusAS.Companion
+PackageVersion: 2.4.2.4911
```
Comment (Trenly) [inline manifests/b/BitfocusAS/Companion/2.4.2.4911/BitfocusAS.Companion.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.4.2
```

File: manifests/b/BitfocusAS/Companion/2.4.2.4911/BitfocusAS.Companion.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: BitfocusAS.Companion
+PackageVersion: 2.4.2.4911
```
Comment (Trenly) [inline manifests/b/BitfocusAS/Companion/2.4.2.4911/BitfocusAS.Companion.yaml:5]:
```suggestion
PackageVersion: 2.4.2
```

**Review by Trenly — CHANGES_REQUESTED** [review]
```
--> Comparing ARP Entries

DisplayName     DisplayVersion Publisher   ProductCode
-----------     -------------- ---------   -----------
Companion 2.4.2 2.4.2          Bitfocus AS 85f8fb94-3357-50b8-9139-7a9cfe700367
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Please resolve https://github.com/microsoft/winget-pkgs/pull/95704#pullrequestreview-1291225775 and sign the CLA as per instructions stated here https://github.com/microsoft/winget-pkgs/pull/95704#issuecomment-1422426313

**Comment by ItzLevvie** [comment]
Close with reason: Stale PR;

**Comment by ItzLevvie** [comment]
[Policy] Needs-CLA

**Pull request closed without merging**